### PR TITLE
Add post deleting. Closes #49

### LINKED ISSUE #49: Deleting posts
I know there's a way to delete a topic, but is there a way to only delete specific posts in the topic?

### PR DESCRIPTION
**Pull request by SunDwarf**

### CODE REVIEW

File: routes/v1/posts.js [hunk 1]
```diff
@@ -10,26 +10,32 @@ var posts = require.main.require('./src/posts'),
...
-		if (req.body.handle) { payload.handle = req.body.handle; }
-		if (req.body.title) { payload.title = req.body.title; }
-		if (req.body.topic_thumb) { payload.options.topic_thumb = req.body.topic_thumb; }
-		if (req.body.tags) { payload.options.tags = req.body.tags; }
-
-		posts.edit(payload, function(err) {
-			errorHandler.handle(err, res);
-		});
+	app.route('/:pid')
+		.post(apiMiddleware.requireUser, function(req, res) {
```
Comment (julianlam) [inline routes/v1/posts.js]: The HTTP verb seems to have changed here :wink: 

Updating assets should be via `PUT`, new additions via `POST`

Comment (SunDwarf) [inline routes/v1/posts.js]: Oh, whoops, didn't even notice that.

Comment (SunDwarf) [inline routes/v1/posts.js]: There, changed it.

**Comment by julianlam** [comment]
Thanks @SunDwarf, can you also please edit `api/v1/README.md` to add the new route?

**Comment by SunDwarf** [comment]
Added

**Comment by julianlam** [comment]
v4.1.0

**Pull request merged**